### PR TITLE
HZN-1419: Wire support for Priority in EventDefinition and UeiMatch

### PR DESCRIPTION
**Pull request by smith-opennms**
This PR adds Priority to the Event and UeiMatch objects and the matching schemas.

### CODE REVIEW

File: xml/src/main/java/org/opennms/integration/api/xml/schema/eventconf/Event.java [hunk 1]
```diff
@@ -61,6 +61,12 @@
     @XmlElement(name="uei", required=true)
     private String m_uei;
 
+    /**
+     * The Priority of the Event definition. Higher number has higher priority.
+     */
+    @XmlElement(name = "priority", required = false)
+    private Integer m_priority;
```
Comment (cgorantla) [inline xml/src/main/java/org/opennms/integration/api/xml/schema/eventconf/Event.java]: missing `m_priority` in equals() and hashCode()

File: xml/src/main/java/org/opennms/integration/api/xml/schema/syslog/UeiMatch.java [hunk 2]
```diff
@@ -103,6 +103,12 @@
     @XmlElement(name = "uei", required = true)
     private String m_uei;
 
+    /**
+     * The Priority of the UeiMatch definition. Higher number has higher priority.
+     */
+    @XmlElement(name = "priority", required = false)
+    private Integer m_priority;
```
Comment (cgorantla) [inline xml/src/main/java/org/opennms/integration/api/xml/schema/syslog/UeiMatch.java]: missing `m_priority` in equals() and hashCode()

File: xml/src/main/java/org/opennms/integration/api/xml/schema/eventconf/Event.java [hunk 3]
```diff
@@ -61,6 +61,12 @@
...
+
     /**
      * A human readable name used to identify an event in the web ui
```
Comment (mattixtech) [inline xml/src/main/java/org/opennms/integration/api/xml/schema/eventconf/Event.java]:
```suggestion
     * A human readable name used to identify an event in the web ui.
```

File: xml/src/main/java/org/opennms/integration/api/xml/schema/syslog/UeiMatch.java [hunk 4]
```diff
@@ -103,6 +103,12 @@
...
+
     /**
      * For regex matches, assign the value of a matching group
      *  to a named event parameter
```
Comment (mattixtech) [inline xml/src/main/java/org/opennms/integration/api/xml/schema/syslog/UeiMatch.java]:
```suggestion
     *  to a named event parameter.
```

File: xml/src/main/java/org/opennms/integration/api/xml/schema/eventconf/Event.java (hunk 1 above)
Comment (smith-opennms) [reply xml/src/main/java/org/opennms/integration/api/xml/schema/eventconf/Event.java]: yup. thanks for both.

**Pull request merged**